Cargo.lock

```diff
@@ -14,9 +14,9 @@ dependencies = [
 
 [[package]]
 name = "addr2line"
```

Comment:
Is this a `cargo update` that slipped through?

Reply:
you mean that i should remove not related to this change update of libs or should not update cargo at all?

Reply:
I would rather do this in another pr